libkbfs/prefetcher_test.go

```diff
@@ -109,15 +109,24 @@ func TestPrefetcherIndirectFileBlock(t *testing.T) {
 	<-q.Prefetcher().Shutdown()
 
 	t.Log("Ensure that the prefetched blocks are in the cache.")
-	block, err = config.BlockCache().Get(ptr1)
+	block, hasPrefetched, lifetime, err :=
+		config.BlockCache().GetWithPrefetch(ptr1)
 	require.NoError(t, err)
 	require.Equal(t, block1, block)
-	block, err = config.BlockCache().Get(ptrs[0].BlockPointer)
+	require.True(t, hasPrefetched)
+	require.Equal(t, TransientEntry, lifetime)
```

Comment:
Would be nice to make a helper method to check the 4 return values of `GetWithPrefetch` in one go, but I don't care much either way.